File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { FreeMusicApi2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await FreeMusicApi2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- FreeMusicApi2 SDK exists test

local sdk = require("free-music-api2_sdk")

describe("FreeMusicApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
